pr_toolbox/quantum/results/frequencies.py

```diff
@@ -38,27 +60,61 @@ def map_counts(counts: Counts, mapper: Callable) -> Counts:
     return Counts(counts_dict)
 
 
-def bitflip_counts(counts: Counts, bitflips: int) -> Counts:
-    """Flip readout bits in counts according to the input bitflips (int encoded).
+def map_quasi_dists(quasi_dists: QuasiDistribution, mapper: Callable) -> QuasiDistribution:
+    """Map quasi-distributions by reassigning keys according to input callable.
 
     Args:
-        counts: the counts to process.
+        quasi_dists: the quasi-distributions to process.
+        mapper: the callable to map readout bits (i.e. counts keys).
+
+    Returns:
+        New QuasiDistribution with readout bits mapped according to input callable.
+    """
+    counts_dict: dict[int, float] = defaultdict(lambda: 0)
```

Comment:
This line is copied from the above code but the variable name not updated:
```suggestion
    frequencies_dict: dict[int, float] = defaultdict(lambda: 0)
```